docs/releasenotes/release-1.126.md

```diff
@@ -1,34 +1,39 @@
 # v1.126.0
 
-** This version is not yet released; this document is gathering release notes for the future release **
-
-* ...
-
-* Special shout-outs to ... for their contributions to this release.
-TODO: list contributors with `git log v1.125.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+* Special shout-outs to @acpana, @Camila-B, @cheftako, @ericpang777, @gemmahou, @himanikh, @jasonvigil, @jingyih, @jsoref, @justinsb, @maqiuyujoyce, @nb-goog, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
 ## Announcement 
 
-* Promoted `ControllerReconciler` and `NamespacedControllerReconciler` CRDs from v1alpha1 to v1beta1.
+* Config Connector system management CRDs `ControllerReconciler` and `NamespacedControllerReconciler` are promoted to Beta. See how to [configure the Controller manager rate limit](https://cloud.google.com/config-connector/docs/how-to/customize-controller-manager-rate-limit).
 
 ## New Beta Resources (Direct Reconciler):
 
-* `Placeholder`
+* [`BigQueryTransformConfig`](https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs)
```

Comment:
The resource is called `BigQueryDataTransferConfig`.

Reply:
Thanks!